webextensions/sidebar/components/TSTCloseBoxElement.js

```diff
@@ -0,0 +1,47 @@
+const NORMAL_TOOLTIP = 'tab_closebox_tab_tooltip';
+const MULTISELECTED_TOOLTIP = 'tab_closebox_tab_tooltip_multiselected';
+const TREE_TOOLTIP = 'tab_closebox_tree_tooltip';
+
+export const CloseBoxTooltipType = Object.freeze({
+  Normal: 'normal',
+  MultiSelected: 'multiselected',
+  Tree: 'tree',
```

Comment:
> Is the PascalCase common for a usecase like this? (Sorry I'm not familiar to common coding styles in WebComponents.)

Ah, I wrote them to reduce an exported object and imagined the style of TypeScript's _enum_ feature.
Should I change to UPPER_CASE?

Reply:
OK, I've understood the context. I'm OK to use PascalCase for enum-like codes in this project, for a future migration to TypeScript (I don't know I can do it in near future...) And (this is off topic) with the same reason other enum-like constants in this project should be re-defined in this style for consistency - it will be separated to another PR.